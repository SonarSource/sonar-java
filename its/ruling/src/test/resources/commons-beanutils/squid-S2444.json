{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/ConvertUtilsBean.java':[
427,
],
}
