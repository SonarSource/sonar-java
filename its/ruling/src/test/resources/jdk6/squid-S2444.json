{
'jdk6:java/awt/AWTKeyStroke.java':[
581,
],
'jdk6:java/awt/KeyboardFocusManager.java':[
192,
3014,
],
'jdk6:java/awt/Toolkit.java':[
418,
673,
],
'jdk6:java/awt/datatransfer/DataFlavor.java':[
653,
],
'jdk6:java/awt/dnd/DragSourceContext.java':[
537,
],
'jdk6:java/net/DatagramSocket.java':[
297,
304,
],
'jdk6:java/net/URLConnection.java':[
330,
],
'jdk6:java/nio/Bits.java':[
561,
562,
],
'jdk6:java/security/Security.java':[
59,
98,
],
'jdk6:java/sql/DriverManager.java':[
127,
128,
432,
434,
436,
],
'jdk6:java/util/Locale.java':[
402,
],
'jdk6:java/util/jar/JarFile.java':[
499,
516,
],
}
