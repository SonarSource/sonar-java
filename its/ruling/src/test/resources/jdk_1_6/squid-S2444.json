{
'jdk_1_6:java/awt/AWTKeyStroke.java':[
581,
],
'jdk_1_6:java/awt/EventDispatchThread.java':[
363,
366,
380,
],
'jdk_1_6:java/awt/GraphicsConfiguration.java':[
395,
410,
],
'jdk_1_6:java/awt/GraphicsEnvironment.java':[
133,
133,
136,
136,
141,
143,
],
'jdk_1_6:java/awt/KeyboardFocusManager.java':[
192,
3014,
],
'jdk_1_6:java/awt/Toolkit.java':[
418,
673,
859,
861,
],
'jdk_1_6:java/awt/datatransfer/DataFlavor.java':[
653,
],
'jdk_1_6:java/awt/dnd/DragSourceContext.java':[
537,
],
'jdk_1_6:java/awt/image/ColorModel.java':[
219,
1696,
1719,
1737,
1756,
1811,
1849,
1898,
1930,
],
'jdk_1_6:java/net/DatagramSocket.java':[
297,
304,
],
'jdk_1_6:java/net/InetAddress.java':[
778,
],
'jdk_1_6:java/net/ProxySelector.java':[
98,
],
'jdk_1_6:java/net/URLConnection.java':[
330,
],
'jdk_1_6:java/nio/Bits.java':[
561,
562,
],
'jdk_1_6:java/nio/channels/spi/SelectorProvider.java':[
79,
99,
157,
],
'jdk_1_6:java/nio/charset/Charset.java':[
401,
],
'jdk_1_6:java/security/AccessControlContext.java':[
111,
],
'jdk_1_6:java/security/Security.java':[
59,
98,
],
'jdk_1_6:java/sql/DriverManager.java':[
127,
128,
432,
434,
436,
],
'jdk_1_6:java/util/Collections.java':[
409,
],
'jdk_1_6:java/util/Locale.java':[
402,
],
'jdk_1_6:java/util/jar/JarFile.java':[
499,
516,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
91,
111,
112,
154,
160,
181,
182,
],
}
