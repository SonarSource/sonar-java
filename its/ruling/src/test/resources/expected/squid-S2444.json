{
'project:com/sun/corba/se/impl/activation/ProcessMonitorThread.java':[
96,
],
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
651,
1604,
],
'project:com/sun/corba/se/impl/javax/rmi/CORBA/Util.java':[
136,
],
'project:com/sun/corba/se/impl/naming/cosnaming/NamingUtils.java':[
85,
104,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/ServantManagerImpl.java':[
208,
],
'project:com/sun/corba/se/impl/orbutil/CorbaResourceUtil.java':[
86,
],
'project:com/sun/corba/se/impl/orbutil/ORBUtility.java':[
108,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClassUtil_1_3.java':[
522,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
625,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
1766,
1793,
],
'project:com/sun/imageio/plugins/jpeg/JPEG.java':[
225,
],
'project:com/sun/imageio/stream/StreamCloser.java':[
96,
],
'project:com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java':[
1470,
1564,
],
'project:com/sun/java/swing/plaf/gtk/GTKStyle.java':[
977,
],
'project:com/sun/java/swing/plaf/gtk/Metacity.java':[
118,
519,
1529,
],
'project:com/sun/java/swing/plaf/motif/MotifIconFactory.java':[
67,
74,
81,
88,
],
'project:com/sun/java/swing/plaf/motif/MotifTreeUI.java':[
101,
102,
103,
104,
],
'project:com/sun/java/swing/plaf/windows/WindowsIconFactory.java':[
72,
79,
86,
93,
100,
107,
114,
130,
137,
144,
151,
158,
],
'project:com/sun/java/swing/plaf/windows/WindowsPopupMenuUI.java':[
74,
],
'project:com/sun/java/swing/plaf/windows/WindowsRootPaneUI.java':[
152,
153,
176,
177,
],
'project:com/sun/java/swing/plaf/windows/WindowsTabbedPaneUI.java':[
72,
78,
],
'project:com/sun/jmx/snmp/SnmpOid.java':[
541,
],
'project:com/sun/jmx/snmp/defaults/DefaultPaths.java':[
101,
155,
209,
224,
],
'project:com/sun/org/apache/bcel/internal/Repository.java':[
88,
],
'project:com/sun/org/apache/bcel/internal/generic/BranchHandle.java':[
89,
101,
],
'project:com/sun/org/apache/bcel/internal/generic/Instruction.java':[
299,
],
'project:com/sun/org/apache/bcel/internal/generic/InstructionHandle.java':[
135,
173,
],
'project:com/sun/org/apache/bcel/internal/util/Class2HTML.java':[
107,
108,
113,
115,
],
'project:com/sun/org/apache/xalan/internal/lib/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/res/XSLMessages.java':[
57,
80,
],
'project:com/sun/org/apache/xalan/internal/xslt/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/BasisLibrary.java':[
1334,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMMessageFormatter.java':[
47,
125,
126,
127,
129,
130,
131,
140,
],
'project:com/sun/org/apache/xerces/internal/dom/ObjectFactory.java':[
177,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/ObjectFactory.java':[
177,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/CaseInsensitiveMap.java':[
65,
105,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
72,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDComplexTypeTraverser.java':[
90,
98,
],
'project:com/sun/org/apache/xerces/internal/parsers/ObjectFactory.java':[
175,
],
'project:com/sun/org/apache/xerces/internal/xinclude/ObjectFactory.java':[
175,
],
'project:com/sun/org/apache/xml/internal/dtm/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xml/internal/res/XMLMessages.java':[
86,
],
'project:com/sun/org/apache/xml/internal/resolver/CatalogManager.java':[
704,
727,
],
'project:com/sun/org/apache/xml/internal/security/algorithms/JCEMapper.java':[
66,
67,
148,
],
'project:com/sun/org/apache/xml/internal/security/keys/keyresolver/KeyResolver.java':[
95,
184,
],
'project:com/sun/org/apache/xml/internal/security/transforms/Transform.java':[
212,
],
'project:com/sun/org/apache/xml/internal/security/utils/CachedXPathFuncHereAPI.java':[
443,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/ResourceResolver.java':[
126,
],
'project:com/sun/org/apache/xml/internal/serialize/HTMLdtd.java':[
375,
376,
],
'project:com/sun/org/apache/xml/internal/serialize/ObjectFactory.java':[
175,
],
'project:com/sun/org/apache/xml/internal/serializer/ObjectFactory.java':[
292,
],
'project:com/sun/org/apache/xml/internal/serializer/OutputPropertiesFactory.java':[
280,
291,
310,
],
'project:com/sun/org/apache/xml/internal/utils/ObjectFactory.java':[
288,
],
'project:com/sun/org/apache/xpath/internal/compiler/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xpath/internal/functions/ObjectFactory.java':[
293,
],
'project:com/sun/org/apache/xpath/internal/jaxp/XPathExpressionImpl.java':[
289,
293,
],
'project:com/sun/org/apache/xpath/internal/jaxp/XPathImpl.java':[
187,
],
'project:com/sun/org/apache/xpath/internal/res/XPATHMessages.java':[
55,
78,
],
'project:java/awt/AWTKeyStroke.java':[
600,
],
'project:java/awt/GraphicsConfiguration.java':[
418,
433,
],
'project:java/awt/GraphicsEnvironment.java':[
171,
171,
174,
174,
179,
181,
],
'project:java/awt/KeyboardFocusManager.java':[
249,
2999,
],
'project:java/awt/Toolkit.java':[
434,
709,
882,
884,
],
'project:java/awt/datatransfer/DataFlavor.java':[
670,
],
'project:java/awt/dnd/DragSourceContext.java':[
573,
],
'project:java/awt/image/ColorModel.java':[
237,
1714,
1737,
1755,
1774,
1830,
1869,
1919,
1953,
],
'project:java/lang/invoke/Invokers.java':[
142,
],
'project:java/lang/invoke/MethodHandleImpl.java':[
1047,
1252,
],
'project:java/lang/management/PlatformComponent.java':[
339,
],
'project:java/net/InetAddress.java':[
804,
],
'project:java/net/ProxySelector.java':[
119,
],
'project:java/net/URLConnection.java':[
347,
],
'project:java/nio/channels/spi/SelectorProvider.java':[
94,
117,
175,
],
'project:java/nio/charset/Charset.java':[
439,
],
'project:java/security/AccessControlContext.java':[
99,
],
'project:java/security/Security.java':[
76,
125,
],
'project:java/util/Collections.java':[
468,
],
'project:java/util/Locale.java':[
809,
831,
834,
],
'project:java/util/jar/JarFile.java':[
517,
534,
],
'project:java/util/prefs/FileSystemPreferences.java':[
104,
124,
125,
167,
173,
194,
195,
],
'project:javax/swing/Autoscroller.java':[
91,
92,
101,
121,
122,
123,
],
'project:javax/swing/BorderFactory.java':[
210,
227,
343,
755,
],
'project:javax/swing/BufferStrategyPaintManager.java':[
172,
178,
],
'project:javax/swing/KeyboardManager.java':[
84,
],
'project:javax/swing/TransferHandler.java':[
742,
],
'project:javax/swing/border/LineBorder.java':[
66,
75,
],
'project:javax/swing/filechooser/FileSystemView.java':[
79,
86,
99,
],
'project:javax/swing/plaf/BorderUIResource.java':[
71,
78,
85,
92,
],
'project:javax/swing/plaf/basic/BasicDesktopPaneUI.java':[
360,
361,
364,
371,
374,
],
'project:javax/swing/plaf/basic/BasicHTML.java':[
243,
244,
288,
],
'project:javax/swing/plaf/basic/BasicIconFactory.java':[
65,
72,
79,
86,
93,
100,
107,
114,
],
'project:javax/swing/plaf/basic/BasicInternalFrameUI.java':[
530,
],
'project:javax/swing/plaf/basic/BasicPanelUI.java':[
48,
],
'project:javax/swing/plaf/basic/BasicRadioButtonUI.java':[
256,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
213,
216,
219,
236,
237,
238,
],
'project:javax/swing/plaf/basic/BasicToolTipUI.java':[
120,
],
'project:javax/swing/plaf/basic/BasicViewportUI.java':[
48,
],
'project:javax/swing/plaf/metal/MetalBorders.java':[
774,
789,
804,
885,
],
'project:javax/swing/plaf/metal/MetalIconFactory.java':[
98,
105,
112,
119,
126,
141,
156,
167,
174,
181,
192,
207,
224,
231,
238,
246,
258,
2264,
2268,
2373,
2377,
],
'project:javax/swing/plaf/metal/MetalScrollBarUI.java':[
107,
108,
109,
110,
111,
112,
],
'project:javax/swing/plaf/metal/MetalScrollButton.java':[
65,
66,
],
'project:javax/swing/plaf/metal/MetalSliderUI.java':[
112,
112,
114,
114,
119,
120,
121,
],
'project:javax/swing/plaf/metal/MetalToolBarUI.java':[
176,
236,
],
'project:javax/swing/plaf/metal/MetalTreeUI.java':[
113,
],
'project:javax/swing/plaf/synth/SynthLookAndFeel.java':[
125,
155,
],
'project:javax/swing/plaf/synth/SynthSplitPaneUI.java':[
103,
111,
],
'project:javax/swing/text/AbstractDocument.java':[
134,
136,
],
'project:javax/swing/text/DefaultCaret.java':[
386,
426,
],
'project:javax/swing/text/GlyphView.java':[
266,
],
'project:javax/swing/text/JTextComponent.java':[
1527,
4961,
],
'project:javax/swing/text/ParagraphView.java':[
69,
71,
],
'project:javax/swing/text/StyleContext.java':[
75,
676,
677,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
595,
],
'project:javax/swing/text/html/Map.java':[
494,
],
'project:javax/xml/bind/DatatypeConverter.java':[
135,
],
'project:org/xml/sax/helpers/XMLReaderFactory.java':[
148,
],
}
